### PR TITLE
Applied conventions on testing/1.7.0

### PR DESCRIPTION
**Pull request by madebr**
Hello,

`bincrafters-conventions` and `conan-readme-generator` were executed on the branch testing/1.7.0

### CODE REVIEW

File: conanfile.py [hunk 1]
```diff
@@ -10,24 +10,27 @@ class OpenH264Conan(ConanFile):
     name = "openh264"
     version = "1.7.0"
     url = "https://github.com/bincrafters/conan-openh264"
+    author = "bincrafters <bincrafters@gmail.com>"
     homepage = 'http://www.openh264.org/'
     description = "Open Source H.264 Codec"
-    license = "BSD 2-Clause"
+    topics = "conan", "h264", "codec", "video", "compression",
```
Comment (Croydon) [inline conanfile.py]: once again:

please use braces in the future `topics = (`

File: conanfile.py [hunk 2]
```diff
@@ -10,24 +10,27 @@ class OpenH264Conan(ConanFile):
     name = "openh264"
     version = "1.7.0"
     url = "https://github.com/bincrafters/conan-openh264"
+    author = "bincrafters <bincrafters@gmail.com>"
```
Comment (Croydon) [inline conanfile.py]: uppercase `Bincrafters`

File: conanfile.py (hunk 1 above)
Comment (madebr) [reply conanfile.py]: ok, is this checked in bincrafters-conventions?

File: conanfile.py (hunk 2 above)
Comment (madebr) [reply conanfile.py]: ok, is this checked in bincrafters-conventions?

**Pull request merged**